### DISCUSSION TITLE
How to get the current member name of a workspace?

### DISCUSSION THREAD

**Discussion by augustocdias** [sagiegurari/cargo-make#631] [Q&A] [answered]
Is there a way to get the name of the member within a workspace to use as a parameter for a script?

Due to other tools limitations I need to pass the member name as parameter for my test script.

Something like this:

```
[tasks.test]
command = "cargo"
args = ["nextest", "run", "--profile", "${MEMBER_NAME}"]
```

**Comment by sagiegurari** [comment, marked as answer]
ya, its called CARGO_MAKE_CRATE_CURRENT_WORKSPACE_MEMBER
